components/VM_Arm/src/main.c

```diff
@@ -802,7 +804,17 @@ static int generate_fdt(vm_t *vm, void *fdt_ori, void *gen_fdt, int buf_size, si
     }
 
     if (config_set(CONFIG_VM_PCI_SUPPORT)) {
-        err = fdt_generate_vpci_node(vm, pci, gen_fdt, GIC_IRQ_PHANDLE);
+        int gic_offset = fdt_path_offset(fdt_ori, vm->gic_path);
+        if (gic_offset < 0) {
+            ZF_LOGE("Failed to find gic node from path: %s", GIC_NODE_PATH);
```

Comment:
```suggestion
            ZF_LOGE("Failed to find gic node from path: %s", vm->gic_path);
```
this also needs an update then

Reply:
Done.